src/components/Button/Button.js

```diff
@@ -56,7 +58,14 @@ Button.propTypes = {
   className: PropTypes.string,
   disabled: PropTypes.bool,
   small: PropTypes.bool,
-  kind: PropTypes.oneOf(['primary', 'secondary', 'danger', 'ghost']).isRequired,
+  kind: PropTypes.oneOf([
+    'primary',
+    'secondary',
+    'danger',
+    'ghost',
+    'dangerPrimary',
```

Comment:
Should we do `primary--danger`? cc @carbon-design-system/ui am unsure what naming convention we want to do here.

Reply:
`danger--primary` (matches the class) or `primary--danger` works for me

Reply:
How about `danger--primary` since it matches the class right now?

Reply:
updated